build-chef-tarball

```diff
@@ -29,10 +31,11 @@ ruby_url='http://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.1.tar.gz'
 chef_version='12.2.1'
 chef_dir='/opt/chef'
 jobs=4
-while getopts 'c:d:hj:r:v:' option; do
+while getopts 'c:d:ghj:r:v:' option; do
 	case "$option" in
 		c) chef_dir=$OPTARG;;
 		d) cache_dir=$OPTARG;;
+		g) GZ=true; WITHOUT_GMP=--without-gmp;;
```

Comment:
i prefer to avoid capitalized variables in bash as bash is one of the only languages that shares the environmental variable namespace with its own variables.

https://github.com/bahamas10/bash-style-guide#variable-declaration